os/net/bluetooth/bluetooth.c

```diff
@@ -311,6 +312,14 @@ int bt_enable(bt_ready_cb_t cb)
 	return 0;
 }
 
+int bt_rand(void *buf, size_t len)
+{
+	/*To check random.h: generate the random number */
+	/*getrandom(buf, len);*/
```

Comment:
To use getrandom() function, we need to enable below 2 configs.
CONFIG_CRYPTO, CONFIG_CRYPTO_RANDOM_POOL